index.js

```diff
@@ -103,7 +103,29 @@ handleMention = (event) => {
             break;
         case "fortune":
             let fortunes = require('fortune-cookie')
-            web.chat.postMessage(fortunes[Math.floor(Math.random() * fortunes.length)]);
+            postMessage(fortunes[Math.floor(Math.random() * fortunes.length)]);
+            break;
+        case "leaderboard":
+            // Sums up offenses from every day per userId
+            let totalUserOffenses = {};
+            for(var offenses in dailyOffenses) {
```

Comment:
`offenses` here isn't actually the offense object, it is actually the key. So to get the actual nest object you will need to get it from `dailyOffenses[offenses]`. So I would change the variable `offset` to `date` and get the offenses from there

Reply:
Better yet `dailyOffenses[date][userId]` when you want to aggregate the total offenses